dash/frontend/src/app/modules/public/pages/login/login.component.scss

```diff
@@ -32,6 +33,7 @@ div {
                 margin: 5px 0;
                 button{
                   width: 100%;
+                  background-color:  mat.get-color-from-palette(mat.$gray-palette, 300);
```

Comment:
This is likely unnecessary (see comment in the html file)

Reply:
This will not be necessary if we choose to use blue (primary) button.